docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc

```diff
@@ -0,0 +1,69 @@
+// tag::ess[]
+
+// QUESTION: Should we add a warning here about updating to 7.13 and include a
+// link to the upgrade docs? 
```

Comment:
could be nice to cross-link it... and to explicitly note that any Agents connected to pre-7.13 stacks, will be automatically unenrolled upon the upgrade of the stack to 7.13.  And they will nee to all be enrolled against a Fleet Server.

Reply:
I'm going to leave this topic as-is for now until we have upgrade info, then I will add links where they are needed.